lib/api/unittest/CDataFrameAnalyzerFeatureImportanceTest.cc

```diff
@@ -65,6 +65,36 @@ void setupLinearRegressionData(const TStrVec& fieldNames,
     }
 }
 
+void setupRegressionDataWithMissingFeatures(const TStrVec& fieldNames,
+                                            TStrVec& fieldValues,
+                                            api::CDataFrameAnalyzer& analyzer,
+                                            std::size_t rows,
+                                            std::size_t cols) {
+    test::CRandomNumbers rng;
+    auto target = [](const TDoubleVec& regressors) {
+        double result{0.0};
+        for (auto regressor : regressors) {
+            result += regressor;
+        }
+        return core::CStringUtils::typeToStringPrecise(result, core::CIEEE754::E_DoublePrecision);
+    };
+
+    for (std::size_t i = 0; i < rows; ++i) {
+        TDoubleVec regressors;
+        rng.generateUniformSamples(0.0, 10.0, cols - 1, regressors);
+
+        fieldValues[0] = target(regressors);
+        for (std::size_t j = 0; j < regressors.size(); ++j) {
+            double regressor = regressors[j] > 9.0 ? core::CDataFrame::valueOfMissing()
```

Comment:
I think if you want to pass missing as a string you need to pass an empty string, i.e. something like:
```
if (regressors[j] <= 9.0) {
    fieldValues[j + 1] = core::CStringUtils::typeToStringPrecise(
                             regressor, core::CIEEE754::E_DoublePrecision);
}
```
(p.s. The approach you use now may work, but it's non-standard.)

Reply:
Fixed in 4eb14fdd